app/src/store/index.js

```diff
@@ -40,6 +40,7 @@ const globalReducer = getReducerWithLocalStorageSync("root", globalSlice.reducer
   "appTheme",
   // "rules",
   // "desktopSpecificDetails.appsList",
+  "hasConnectedApp",
```

Comment:
Why do we need to create another top level key here?  This can be added to `misc.persist`